docs/source/api/algorithms/xgi.algorithms.centrality.rst

```diff
@@ -10,4 +10,5 @@ xgi.algorithms.centrality
    .. autofunction:: CEC_centrality
    .. autofunction:: HEC_centrality
    .. autofunction:: ZEC_centrality
-   .. autofunction:: node_edge_centrality
\ No newline at end of file
+   .. autofunction:: node_edge_centrality
+   .. autofunction:: vector_centrality
```

Comment:
I know this is probably the canonical name in the reference paper, but it might be too general for a general purpose library like XGI. Perhaps we should consider `line_vector_centrality` to make allusion to the fact that this uses the line graph.